### DISCUSSION TITLE
- move parameter change the creation date of the file.

### DISCUSSION THREAD

**Discussion by islogged** [tfeldmann/organize#289] [General]
Hi,

Your tool is useful, but the parameter "-move" move the files as it told but also change the creation date of the file to the current date/time of the move action.

How to keep the original date of the file when use the -move parameter?
Regards,

**Comment by tfeldmann** [comment]
This might be a bug in the underlying pyfilesystem library, maybe a duplicate of #203

**Reply by islogged** [reply]
> This might be a bug in the underlying pyfilesystem library, maybe a duplicate of https://github.com/tfeldmann/organize/issues/203

Thx for ur answer, anyway i decided to make a powershell script who replace organize for my use and solve this problem.
Thx again...
